core/server/models/user.js

```diff
@@ -554,6 +554,12 @@ User = ghostBookshelf.Model.extend({
                 return bcryptCompare(object.password, user.get('password')).then(function then(matched) {
                     if (!matched) {
                         return Promise.resolve(self.setWarning(user, {validate: false})).then(function then(remaining) {
+                            if (remaining === 0) {
```

Comment:
if you would change `if (level > 4) {` in line `529` to `if (level > 3) {`, then it works without having this extra check here.